README.md

```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
 ## Status and released versions
 
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
-* Release 0.1.0 of the API family is available within the [release-v0.1.0](https://github.com/camaraproject/PopulationDensityData/tree/release-v0.1.0) branch.
-- 0.1.0 Population Density Data API definition **with inline documentation**:
-  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/release-v0.1.0/code/API_definitions/population-density-data.yaml)
-  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.0/code/API_definitions/population-density-data.yaml&nocors)
-  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.0/code/API_definitions/population-density-data.yaml)
+* Release 0.1.1-rc1.1 of the API family is available within the [release-v0.1.1-rc1.1](https://github.com/camaraproject/PopulationDensityData/tree/release-v0.1.1-rc1.1) branch.
```

Comment:
```suggestion
* The pre-release r1.1 contains the version 0.1.1-rc.1 of the Population Density Data API. It is the first release candidate of the patch release v0.1.1. The release tag is [r1.1](https://github.com/camaraproject/PopulationDensityData/tree/r1.1).
```

There is no mandatory text therefore this is just a suggestion. 
Important is just:
* we don't use/create release branches any more
* The release tag is r1.1
* The API Version is v0.1.1-rc.1, not v0.1.1-rc1.1